Chapter1/Zad5.py

```diff
@@ -0,0 +1,38 @@
+'''
+Created on 28 sty 2018
+
+@author: agata.wiewiora
+'''
+
+
+def count_tables_amount (students_amount):
+    table_students = 2 
+    temp = students_amount % table_students
+    if temp > 0:
+        tables_amount = (students_amount // table_students) + 1
+    else:
+        tables_amount = students_amount // table_students
+    return tables_amount
+
+students = {}
```

Comment:
tutaj bardziej pasowałaby lista niż dict, bo nie ma potrzeby żeby rozróżniać klasy wg kluczy, ale to jest wybieganie z materiałem